backend/src/main/java/ca/bc/gov/app/util/HandlerUtils.java

```diff
@@ -0,0 +1,37 @@
+package ca.bc.gov.app.util;
+
+import java.util.Optional;
+import java.util.function.Consumer;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.reactive.function.BodyInserters;
+import org.springframework.web.reactive.function.server.ServerResponse;
+import org.springframework.web.server.ResponseStatusException;
+import reactor.core.publisher.Mono;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class HandlerUtils {
```

Comment:
@paulushcgcj please keep the names in singular, so just leave it as "HandlerUtil". Thanks.